client/devpi/main.py

```diff
@@ -855,6 +855,8 @@ def install(parser):
     parser.add_argument("--venv", action="store", metavar="DIR",
         help="install into specified virtualenv (created on the fly "
              "if none exists).")
+    parser.add_argument("-r", "--requirement", action="store_true",
```

Comment:
i would like to suggest to change this to a collecting list, so people dont get surprised if they use -r more than once or mix requirement files and normal requirements